rescript/get_gtdb.py

```diff
@@ -52,6 +55,12 @@ def _assemble_queries(version='214.1',
     # ^^ Set `base_version` variable becuase number after the decimal is
     # only used for the directory. GTDB trims this off for the actual
     # file names...
+    # GTDB v220 started storing the ssu_reps FASTA files
+    # as 'fna.gz' instead of their usual 'tar.gz'.
+    if version == '220.0':
```

Comment:
should this be >=? Or do you want to wait to react until the next release is out in case they change again?

Reply:
never mind, I realize that this is evaluating a string, so ignore my comment. We can only address this when we have a new version (though we might want to instead check if it is NOT in the list of older versions, otherwise we need to update the list each time a new version is released).